File: Programming Basics/02-Conditional Statements - Exercise/godzillaVsKong.js
function godzillaVsKong(input) {

    let budgetForMovie = Number(input[0]);
    let numberOfExtras = Number(input[1]);
    let clothesPriceForOneExtras = Number(input[2]);

    let decor = budgetForMovie * 0.10;

    if(numberOfExtras >= 150) {
        clothesPriceForOneExtras *= 0.90;
    }

    let priceForTheMovie = numberOfExtras * clothesPriceForOneExtras + decor;

    let total = Math.abs(budgetForMovie - priceForTheMovie);
    
    if(priceForTheMovie > budgetForMovie) {
        console.log("Not enough money!");
        console.log(`Wingard needs ${total.toFixed(2)} leva more.`);
    } else {
        console.log("Action!" );
        console.log(`Wingard starts filming with ${total.toFixed(2)} leva left.`);
    }

    
}

godzillaVsKong(["9587.88",
"222",
"55.68"]);

File: Programming Basics Java/src/ConditionalStatementsExercise/GodzillaVsKong.java
package ConditionalStatementsExercise;

import java.util.Scanner;

public class GodzillaVsKong {
    public static void main(String[] arg) {
        Scanner scan = new Scanner(System.in);

        double movieBudget = Double.parseDouble(scan.nextLine());
        int numberOfPeople = Integer.parseInt(scan.nextLine());
        double priceForClothPerPerson = Double.parseDouble(scan.nextLine());

        double priceForDecors = movieBudget * 0.10;
        double priceForCloths = numberOfPeople * priceForClothPerPerson;

        if (numberOfPeople > 150) {
            priceForCloths *= 0.90;
        }

        double totalExpenses = priceForDecors + priceForCloths;

        double moneyOverOrUnder = Math.abs(movieBudget - totalExpenses);

        if (movieBudget >= totalExpenses) {
            System.out.println("Action!");
            System.out.printf("Wingard starts filming with %.2f leva left.", moneyOverOrUnder);
        } else {
            System.out.println("Not enough money!");
            System.out.printf("Wingard needs %.2f leva more.", moneyOverOrUnder);
        }
    }
}
